go/service/gregor.go

```diff
@@ -96,6 +146,10 @@ func (g *gregorHandler) ShouldRetryOnConnect(err error) bool {
 func (g *gregorHandler) BroadcastMessage(ctx context.Context, m gregor1.Message) error {
 	g.G().Log.Debug("gregor handler: broadcast: %+v", m)
 
+	// Send message to local state machine
+	g.gregorCli.ConsumeMessage(m)
```

Comment:
Yeah that comment is oddly specific, I guess people wanted to do this using that function.